entrypoint.sh

```diff
@@ -71,17 +79,16 @@ initialize_worker_node() {
   echo "Executing: $CMD"
   eval "$CMD"
 
-  # Start sllm-store-server with any additional arguments passed to the script
-  STORAGE_PATH="${STORAGE_PATH:-$DEFAULT_STORAGE_PATH}"
-  echo "Starting sllm-store-server with arguments: -storage_path=$STORAGE_PATH $@"
-  exec sllm-store-server -storage_path=$STORAGE_PATH "$@"
+  initialize_sllm_store "$@"
 }
 
 # Determine the node type and call the appropriate initialization function
 if [ "$MODE" == "HEAD" ]; then
   initialize_head_node "$@"
 elif [ "$MODE" == "WORKER" ]; then
   initialize_worker_node "$@"
+elif [ "$MODE" == "STORE" ]; then
```

Comment:
The entrypoint should support only two modes: "head" and "worker." If it's purely a store, it should be located in the "sllm_store/" directory. Also, why do we need the "sllm worker" environment for a "store-only" container image?